test/unit/tech/html5.test.js

```diff
@@ -131,14 +133,39 @@ QUnit.test('patchCanPlayType patches canplaytype with our function, conditionall
                     'patched canPlayType and function returned from unpatch are equal');
 
   browser.ANDROID_VERSION = oldAV;
+  browser.IS_FIREFOX = oldIsFirefox;
+  Html5.unpatchCanPlayType();
+});
+
+QUnit.test('patchCanPlayType doesn\'t patch canplaytype with our function in Firefox for Android', function(assert) {
+  // the patch runs automatically so we need to first unpatch
+  Html5.unpatchCanPlayType();
+
+  const oldAV = browser.ANDROID_VERSION;
+  const oldIsFirefox = browser.IS_FIREFOX;
+  const video = document.createElement('video');
+  const canPlayType = Html5.TEST_VID.constructor.prototype.canPlayType;
+
+  browser.ANDROID_VERSION = 4.0;
+  browser.IS_FIREFOX = true;
+  Html5.patchCanPlayType();
+
+  assert.strictEqual(video.canPlayType,
+                 canPlayType,
+                 'original canPlayType and patched canPlayType should not be equal');
```

Comment:
Leftover from copy & paste of the other test, fixed in b762008d22b1424e2b16140d6d98e7465b51d1fd.